resources/views/inc/navbar.blade.php

```diff
@@ -22,7 +22,7 @@
                 <a class="nav-link" href="/askme"> &nbsp;  &nbsp; Ask Me</a>
             </li>
             <li class="nav-item">
-                    <a class="nav-link" href="posts/"> &nbsp;  &nbsp; Blog</a>
+                    <a class="nav-link" href="/posts"> &nbsp;  &nbsp; Blog</a>
```

Comment:
This tiny mistake still let you rout between pages however it give you some parameter error such as

Trying to get property 'name' of non-object

that would confuse you.